integration-tests/Package.swift

```diff
@@ -0,0 +1,26 @@
+// swift-tools-version: 6.0
+
+import PackageDescription
+
+let linkerSettings: [LinkerSetting] = [
+    .linkedLibrary("swift_integration_tests"),
+    .unsafeFlags(["-L../target/debug/"])
+]
+
+let package = Package(
+    name: "IntegrationTests",
+    targets: [
+        // The compiled static rust library.
+        .systemLibrary(
+            name: "RustLib"),
+        // The generated Swift wrapper code for the Rust library, plus some
+        // Swift code used by the Rust library.
```

Comment:
Let's be more explicit about exactly what this "generated Swift wrapper code" and "Swift code used by the Rust library is".

A new contributor should be able to read this and know where to look if they want more information.

So:

- clarify that the generated Swift wrapper code is the FFI bindings that `swift-bridge` generates so that the languages can call into each other
- clarify that "Swift code used by the Rust library" is the `swift-bridge` core code that is always generated for all projects, and mention the name of the file that this code gets created in.
  - `crates/swift-bridge-build/src/generate_core.rs`

This level of clarity will help maintainers that are navigating the test suite more quickly understand where to look next when they want to fix something.